coderd/templates.go

```diff
@@ -861,7 +874,7 @@ func (api *API) convertTemplate(
 		autostopRequirementWeeks = 1
 	}
 
-	return codersdk.Template{
+	x := codersdk.Template{
```

Comment:
Can we remove `x` or name it better?

Reply:
This was a mistake, I named the variable so I could target it in a debugger better. Reverted